docs/source/quickstart.mdx

```diff
@@ -85,4 +85,70 @@ tokenizer = AutoTokenizer.from_pretrained(model_id)
 pipe_cls = pipeline("text-classification", model=model, tokenizer=tokenizer)
 text = "He's a dreadful magician."
 outputs = pipe_cls(text)
+```
+
+# Distributed training
+
+Distributed training is implemented by intel oneCCL and MPI.
```

Comment:
will we support GLOO, NCCL and other communication libraries?

Reply:
We'll support GLOO, as it's a default CCL in PyTorch. And we'll not validate NCCL here, since we treat optimum-intel mainly on Intel's stacks.

@jiqing-feng  Suggest to wording as below(just an example):
CPU distributed distillation supports both Gloo and oneCCL as collective communication backend. And for performance seek, Intel recommend to use oneCCL backend.
We will show how to use oneCCL backend-ed distributed distillation as below steps.